### PR TITLE
Prepare files for 2.0.0-beta.2

### PR DESCRIPTION
**Pull request by nicosomb**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | yes
| Translation   | yes
| Fixed tickets | N/A
| License       | MIT

Because we have problems for those who install wallabag via `composer create-project` (no way to upgrade their installation), we considered to use `git clone`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,10 +11,14 @@ More informations on our website: [wallabag.org](https://wallabag.org)
...
-If you don't have it yet, please [install composer](https://getcomposer.org/download/). Then you can install wallabag by executing the following commands:
+If you don't have it yet, please [install composer](https://getcomposer.org/download/) and git.
+Then you can install wallabag by executing the following commands:
 
 ```
-SYMFONY_ENV=prod composer create-project wallabag/wallabag wallabag "2.0.0-beta.1" --no-dev
+git clone git@github.com:wallabag/wallabag.git
+cd wallabag
+git checkout 2.0.0-beta.2
+SYMFONY_ENV=prod composer install --no-dev
```
Comment (j0k3r) [inline README.md]: Could you add `-o` at the end? It'll optimize the autoload.

**Comment by nicosomb** [comment]
With this new installation, the user needs to generate a new github token. 
@wallabag/core do you know if we can avoid this behavior? 

(thank you @chris25640 for your tests)

**Comment by nicosomb** [comment]
Finally back to `composer create-project` with `--keep-vcs` option.

**Comment by nicosomb** [comment]
Finally back to composer create-project with --keep-vcs option.

**Pull request merged**